packages/flutter/test/widgets/banner_test.dart

```diff
@@ -295,4 +295,32 @@ void main() {
       areCreateAndDispose,
     );
   });
+
+  testWidgets('Banner widget respects shadow', (WidgetTester tester) async {
+    debugDisableShadows = false;
+    await tester.pumpWidget(
+      const Directionality(
+           textDirection: TextDirection.ltr,
```

Comment:
```suggestion
        textDirection: TextDirection.ltr,
```

Reply:
I think I [beat you to it](https://github.com/flutter/flutter/pull/155296#discussion_r1774224150) :)